todo_app/utils.py

```diff
@@ -0,0 +1,10 @@
+
+
+def SimpleValidation(*argv):
```

Comment:
Naming convention in Python is `snake_case` for variable/function names and `PascalCase` for class names.

And you rename this to something like `empty_or_whitespace`. My first instinct would have incorrectly been that True means it's valid.